src/main/java/net/imglib2/util/Intervals.java

```diff
@@ -860,6 +860,14 @@ public static boolean equalDimensions( final Dimensions a, final Dimensions b )
 		return true;
 	}
 
+	/**
+	 * Tests whether two intervals have equal dimensions (same size).
+	 */
+	public static boolean equalDimensions( final Interval a, final Interval b )
+	{
+		return equalDimensions((Dimensions) a, (Dimensions) b);
```

Comment:
Tiny style formatting issue with this line: parentheses lack surrounding whitespace.

Reply:
```suggestion
		return equalDimensions( ( Dimensions ) a, ( Dimensions ) b );
```